improver_tests/spotdata/test_SpotLapseRateAdjust.py

```diff
@@ -187,12 +206,21 @@ def test_basic(self):
         self.assertEqual(result.coords(), self.spot_temperature_nearest.coords())
 
     def test_nearest_neighbour_method(self):
-        """Test that the plugin modifies temperatures as expected using the
-        vertical displacements taken from the nearest neighbour method in the
-        neighbour cube."""
+        """Test that the plugin modifies temperatures as expected for both air
+        temperature and feels like temperature cubes, using the vertical
+        displacements taken from the nearest neighbour method in the neighbour cube."""
+
         plugin = SpotLapseRateAdjust()
         expected = np.array([280 + (2 * DALR), 270, 280 - DALR]).astype(np.float32)
 
+        # Air temperature cube
+        result = plugin(
+            self.spot_temperature_nearest, self.neighbour_cube, self.lapse_rate_cube
+        )
+        self.assertArrayEqual(result.data, expected)
+
+        # Feels like temperature cube
```

Comment:
This feels like it ought to be a parametrized pytest as the only difference between the two halves of the test is the cube name. Splitting it into two separate tests here seems unpythonic, so don't do that.

On reflection, given that this test module is in the unittest style, and therefore converting to a parametrized pytest would be awkward, leave this as it is.